.gitignore

```diff
@@ -2,13 +2,17 @@
 .idea/modules.xml
 .idea/*.iml
 .idea/**/workspace.xml
+set_*
+yapf.sh*
 .idea/codeStyles
 
+
 # Project
 project/__pycache__/
 **.pyc
 lib/
 .logs/
+.DS_Store
```

Comment:
btw what is .DS_Store?

Reply:
Please remove this line if we don't need it.

Reply:
It's some hidden file that is automatically generated on Mac and is irrelevant for the application. I'd rather keep it in so that others don't have a random file that isn't pertinent to the project on their development platform.